node/node.go

```diff
@@ -357,6 +382,13 @@ func (n *Node) run(ctx context.Context) (err error) {
 			n.role = certUpdate.Role
 			n.roleCond.Broadcast()
 			n.Unlock()
+
+			// Export the new role.
+			if n.currentRole() == api.NodeRoleManager {
+				nodeManager.Set(1)
+			} else {
+				nodeManager.Set(0)
+			}
```

Comment:
We could simplify this to `nodeManager.Set(certUpdate.Role == ca.ManagerRole)`, but I'm not sure if you find that cleaner.